### ISSUE TITLE
2.4 Transfer Learning & Fine-Tuning.ipynb

### ISSUE THREAD

**Issue by michael135** [ypeleg/Deep-Learning-Keras-Tensorflow-PyCon-Israel-2017#2]
After "re-training" I checked again the preformance on **X_train_lt5** and **X_test_lt5** and it's accuracy decreased from ~99% to ~40%.

I believe in retraining the "old examples" should be partially combined with a new ones...
